quickstarts/runway-quickstart.yml

```diff
@@ -0,0 +1,237 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Deploys an ec2 instance with Runway pre-configured for evaluation
+
+Parameters:
+  OS:
+    Type: String
+    Description: The preferred OS type (Windows/Linux) of the Runway instance
+    AllowedValues:
+    - Windows
+    - Linux
+
+  KeyPair: 
+    Description: The existing EC2 KeyPair to be used to access the Runway instance
+    Type: AWS::EC2::KeyPair::KeyName
+
+  SourceIP: 
+    Description: The egress (public) IPv4 address from which you plan to access your Runway instance.  Hint- https://whatismyip.com .  Specify address only, do not include /CIDR designator, example 157.123.231.123
+    Type: String
+
+  IamRole:
+    Type: String
+    Description: Choose 'auto/admin' to have this CloudFormation template deploy an ADMIN IAM Role for Runway to use to call AWS services.  Choose 'manual' to specify an existing IAM Role with more restrictive permissions.
+    AllowedValues:
+    - auto/admin
+    - manual
+
+  IamRoleName: 
+    Description: If you chose 'manual' for IamRole, specify the name of an existing IAM Role here, otherwise leave as the default value of 'none'
+    Type: String
+    Default: none
+
+
+Conditions: 
+  DeployWindows: !Equals [!Ref OS, Windows]
+  DeployLinux: !Equals [!Ref OS, Linux]
+  DeployIamRole: !Equals [!Ref IamRole, auto/admin]
+
+
+Mappings:
+  AMIs:
+    us-east-2:
```

Comment:
Only thing I would suggest would be adding other regions to your AMI map in the CFN template.

Reply:
@GarisonLotus agreed...see 'cfn mappings' comment a few lines above yours ^^     :)
If I can get this merged I'll add the mappings today...